model/internal/otlp_wrapper.go

```diff
@@ -38,6 +39,58 @@ func MetricsFromOtlp(req *otlpcollectormetrics.ExportMetricsServiceRequest) Metr
 	return MetricsWrapper{req: req}
 }
 
+func intHistogramToHistogram(src *otlpmetrics.Metric_IntHistogram) *otlpmetrics.Metric_Histogram {
+	datapoints := []*otlpmetrics.HistogramDataPoint{}
+	for _, datapoint := range src.IntHistogram.DataPoints {
+		exemplars := []*otlpmetrics.Exemplar{}
+		for _, exemplar := range datapoint.Exemplars {
+			exemplars = append(exemplars, &otlpmetrics.Exemplar{
+				FilteredLabels: exemplar.FilteredLabels,
+				TimeUnixNano:   exemplar.TimeUnixNano,
+				Value: &otlpmetrics.Exemplar_AsDouble{
+					AsDouble: float64(exemplar.Value),
+				},
```

Comment:
It should be AsInt correct?

Reply:
Right, I misinterpreted the comment in the proto about values being translated to float from int64 to include the values of the Exemplar.